docs/testspec/src/18_tpm.rst

```diff
@@ -0,0 +1,468 @@
+   |                        | - persistent key ID of an RSA key                                       |
+   +------------------------+-------------------------------------------------------------------------+
+   | **Expected Output:**   | n/a                                                                     |
+   +------------------------+-------------------------------------------------------------------------+
+   | **Steps:**             | #. Create a TPM2 context and an unauthenticated session                 |
+   |                        |                                                                         |
+   |                        | #. Instantiate the passed persistent RSA key pair using an incorrect    |
+   |                        |    authentication value                                                 |
+   |                        |                                                                         |
+   |                        | #. Check that the signature creatino fail with a "TPM2 Error"           |
+   +------------------------+-------------------------------------------------------------------------+
+
+.. table::
+   :class: longtable
+   :widths: 20 80
+
+   +------------------------+-------------------------------------------------------------------------+
+   | **Test Case No.:**     | TPM-RSA-3                                                               |
+   +========================+=========================================================================+
+   | **Type:**              | Positive/Negative Test                                                  |
+   +------------------------+-------------------------------------------------------------------------+
+   | **Description:**       | Encryption/Decryption of a message using RSA                            |
+   +------------------------+-------------------------------------------------------------------------+
+   | **Preconditions:**     | The TPM 2.0 software emulator is running and provisioned properly       |
+   +------------------------+-------------------------------------------------------------------------+
+   | **Input Values:**      | - TCTI name and config                                                  |
+   |                        | - persistent key ID of an RSA key                                       |
+   +------------------------+-------------------------------------------------------------------------+
+   | **Expected Output:**   | n/a                                                                     |
+   +------------------------+-------------------------------------------------------------------------+
+   | **Steps:**             | #. Create a TPM2 context and an unauthenticated session                 |
+   |                        |                                                                         |
+   |                        | #. Instantiate the passed persistent RSA key pair using an incorrect    |
```

Comment:
incorrect?

Reply:
Copypasta from the previous table. Thanks.